modules/actions/functions.js

```diff
@@ -0,0 +1,151 @@
+import { createAction } from './utils';
+import AppConstants from '../constants';
+import {
+	getFunctions as fetchFunctions,
+	updateFunctions as putFunctions,
+	createFunction as deployFunction,
+	invokeFunction as invokeFunctions,
+	deleteFunction as deleteFunctions,
+	reorderFunction as reorderFunctions,
+	getSingleFunction as getFuncInfo,
+} from '../../utils/app';
+
+export function getFunctions(name = 'default') {
```

Comment:
@ShahAnuj2610 You can directly use `doGet`, `doPost` utility function in actions instead of writing functions at other places unless that function is getting used at somewhere else.